ocfweb/stats/templates/stats/summary.html

```diff
@@ -11,6 +11,7 @@
             <img class="img-responsive" src="{% url 'daily_graph_image' %}" />
             <img class="img-responsive" src="{% url 'session_count_image' %}" />
             <img class="img-responsive" src="{% url 'session_length_image' %}" />
+            <img class="img-responsive" src="{% url 'daily_job_image' %}" />
```

Comment:
I think this should go under the printing tab (at [ocfweb/stats/templates/stats/printing.html](https://github.com/ocf/ocfweb/blob/master/ocfweb/stats/templates/stats/printing.html)) instead of the main stats page, since it has more to do with printing.